cognite_toolkit/_cdf_tk/warnings.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
```

Comment:
The module should not be called `warnings` as that is the name of a built in module in python